packages/open-next/package.json

```diff
@@ -17,7 +17,8 @@
     "dev": "tsc -w"
   },
   "exports": {
-    "./*": "./dist/*"
+    "default": "./dist/*",
+    "types": "./dist/*.d.ts"
   },
   "typesVersions": {
```

Comment:
Is `typesVersion` useful here? 
Maybe we can drop that while we are updating this file

Reply:
You're right here too. Given the version is wildcard, the exports types cover this.